_locales/nl/messages.json

```diff
@@ -0,0 +1,182 @@
+    "description": "The description for the extension, used in the manifest."
+  },
+  "fillUsernamePassDescription": {
+    "message": "Vul uw gebruikersnaam en paswoord vanuit het huidige actieve invoerveld",
+    "description": "The description for the fill username/password hotkey"
+  },
+  "fillPassDescription": {
+    "message": "Vul uw paswoord in het hduige active invoerveld",
+    "description": "The description for the fill password hotkey"
+  },
+  "SSsetupCancelled": {
+    "message": "Secure Storage Setup werd geannuleerd, Keywi kan niet functioneren zonder deze stap. Open de instellingen om dit opnieuw in te stellen.",
+    "description": "Displayed when the setup for the Secure Storage is cancelled."
+  },
+  "SSunlockCancelled": {
+    "message": "Secure Storage ontgrendelen werd geannuleerd, Keywi kan niet functioneren zonder deze stap.",
+    "description": "Displayed when the unlock for the Secure Storage is cancelled."
+  },
+  "SSwrongUnlockKey": {
+    "message": "U heeft een foute key ingevoerd.",
+    "description": "Alert when the wrong key is given at Secure Storage unlock time"
+  },
+  "contextFillUserPass": {
+    "message": "Vul gebruikersnaam and paswoord in",
+    "description": "Context menu to fill username and password"
+  },
+  "contextFillUser": {
+    "message": "Vul gebruikersnaam in",
+    "description": "Context menu to fill username"
+  },
+  "contextFillPass": {
+    "message": "Vul paswoord in",
+    "description": "Context menu to fill password"
+  },
+  "noPassFieldFound": {
+    "message": "Geen paswoord invoerveld gevonden, enkel de gebruikersnaam word ingevuld",
+    "description": "Displayed when no password field was found when filling username and password"
+  },
+  "otherDBOpen": {
+    "message": "Keywi is reeds geassocieerd met een database, maar een andere database is geopend.",
+    "description": "Displayed at initialization if another database than the one keywi is associated with is open"
+  },
+  "cannotConnect": {
+    "message": "Keywi kan niet verbinden met Keepass, is uw Keepass database geopend?",
+    "description": "Displayed when keywi cannot connect to the Keepass database"
+  },
+  "noPassFound": {
+    "message": "Geen paswoorden gevonden",
+    "description": "Displayed when keywi cannot find any suitable passwords"
+  },
+  "noLogins": {
+    "message": "Er was een probleem bij het ophalen van uw credentials, heeft u Keywi geassocieerd met de Keepass database?",
+    "description": "Displayed when Keepass doesn't give access"
+  },
+  "assocFailed": {
+    "message": "Er ging iets mis tijdens de associatie met Keepass",
+    "description": "Displayed when something goes wrong during association"
+  },
+  "statusSSunlocked": {
+    "message": "Secure storage ontgrendeld",
+    "description": "is Secure Storage unlocked?"
+  },
+  "statusDBassoc": {
+    "message": "Keepass database geassocieerd",
```

Comment:
'geassocieerd met ...' ?

Reply:
![selectie_704](https://user-images.githubusercontent.com/2996275/29747832-178a2cb6-8b07-11e7-95f8-b4bd728dfff9.png)
This is text is shown on the options page, I think it's okay like this?